app/jobs/activities/send_webhook_job.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Activities
+  class SendWebhookJob < ApplicationJob
+    queue_as :high_priority
+
+    def perform(webhook, activity)
+      Activities::ActivityWebhookService.new(webhook, activity).send_webhook
+    end
+  end
+end
```

Comment:
This could maybe just be a generalized webhook job, but I made it separate in case we maybe have some lower priority webhooks in the future.